File: cpp/HardwareKomponen.cpp
#pragma once
#include <iostream>
#include <string>


using namespace std;


class HardwareKomponen {
private:
    string model;
    int konsumsiDayaWatt;
public:
    HardwareKomponen()
    {

    }


    HardwareKomponen(string model, int konsumsiDayaWatt)
    {
        this->model = model;
        this->konsumsiDayaWatt = konsumsiDayaWatt;
    }

    void setModel(string model)
    {
        this->model = model;
    }

    void setKonsumsiDayaWatt(int konsumsiDayaWatt)
    {
        this->konsumsiDayaWatt = konsumsiDayaWatt;
    }


    string getModel()
    {
        return model;
    }
    int getKonsumsiDayaWatt()
    {
        return konsumsiDayaWatt;
    }


    ~HardwareKomponen()
    {




    }
};

File: py/HardwareKomponen.py
# HardwareKomponen.py
class HardwareKomponen:
    def __init__(self, model="", konsumsiDayaWatt=0):
        self.model = model
        self.konsumsiDayaWatt = konsumsiDayaWatt
    
    def setModel(self, model):
        self.model = model
    
    def setKonsumsiDayaWatt(self, konsumsiDayaWatt):
        self.konsumsiDayaWatt = konsumsiDayaWatt
    
    def getModel(self):
        return self.model
    
    def getKonsumsiDayaWatt(self):
        return self.konsumsiDayaWatt